### PR TITLE
[MINOR] Fix typos

### PR DESCRIPTION
**Pull request by shuoer86**

### CODE REVIEW

**Comment by srowen** [comment]
Can you find more while you're at it?

**Review by srowen — APPROVED** [review]

File: core/src/main/scala/org/apache/spark/storage/DiskBlockManager.scala [hunk 1]
```diff
@@ -79,7 +79,7 @@ private[spark] class DiskBlockManager(
   private val shutdownHook = addShutdownHook()
 
   // If either of these features are enabled, we must change permissions on block manager
-  // directories and files to accomodate the shuffle service deleting files in a secure environment.
+  // directories and files to accommodate the shuffle service deleting files in a secure environment.
```
Comment (srowen) [inline core/src/main/scala/org/apache/spark/storage/DiskBlockManager.scala:82]: See the test failure results. This line is too long now

`[error] /home/runner/work/spark/spark/core/src/main/scala/org/apache/spark/storage/DiskBlockManager.scala:82: File line length exceeds 100 characters`

Comment (shuoer86) [reply core/src/main/scala/org/apache/spark/storage/DiskBlockManager.scala]: done

**Review by srowen — APPROVED** [review]

**Comment by srowen** [comment]
Merged to master

**Pull request closed without merging**